drivers/SmartThings/sonos/src/api/sonos_websocket_router.lua

```diff
@@ -10,14 +10,14 @@ local CloseCode = require "lustre.frame.close".CloseCode
 local lb_utils = require "lunchbox.util"
 local st_utils = require "st.utils"
 
+-- override module-wide logging to reduce chattiness
```

Comment:
nit: Since we're not overriding anything in this module, I'm okay with getting rid of this and directly `require`ing `"log"` as `log` instead of `old_log`. But I'm also okay leaving this in as a hook to adjust later if needed. Your call.

Reply:
Good point - removed in 80f53ff47cdd202d26cdef7adba4a9c75b8b8e84